messagingpush/src/main/java/io/customer/messagingpush/CustomerIOPushNotificationHandler.kt

```diff
@@ -62,7 +65,7 @@ internal class CustomerIOPushNotificationHandler(private val remoteMessage: Remo
     ): Boolean {
         // Check if message contains a data payload.
         if (bundle.isEmpty) {
-            Log.d(TAG, "Message data payload: $bundle")
+            logger.debug("Message data payload: $bundle")
```

Comment:
Currently, the `Logger` doesn't differentiate logs from 1 file to another. So, I recommend log statements to be more descriptive then `message data payload` because maybe some other file in the SDK also uses `message data payload`. 

```suggestion
            logger.debug("Push message received is empty.")            
```

Then outside of this `if` statement where the bundle is not empty:
```kotlin
logger.debug("Handling push message. Bundle: $bundle")
```